consensus.go

```diff
@@ -0,0 +1,236 @@
+package dcrlibwallet
+
+import (
+	"fmt"
+	"sort"
+	"time"
+
+	"decred.org/dcrwallet/v2/errors"
+	w "decred.org/dcrwallet/v2/wallet"
+	"github.com/decred/dcrd/chaincfg/chainhash"
+	"github.com/decred/dcrd/chaincfg/v3"
+)
+
+// GetVoteChoices returns configured vote preferences for each agenda of
+// all supported stake versions, but only preferences saved for the
+// current stake version agendas for the wallet with the specified walletID.
+func (mw *MultiWallet) GetVoteChoices(hash string, walletID int) (*GetVoteChoicesResult, error) {
```

Comment:
Since this is wallet-specific, and does not really require access to the MultiWallet instance, I recommend making this and similar methods `(wallet *Wallet)` methods. Callers who do not have the desired wallet instance handy but have a `mw` instance and a walletID, can call `mw.WalletWithID` before calling the wallet method. I expect there'd be instances where the wallet instance is already available and can be used directly.